README.md

```diff
@@ -97,22 +116,19 @@ $ fh list releases nixos/nixpkgs
 ...
 ```
 
-### Listing organizations and flakes
+### Listing organizations, flakes, and versions
 
-`fh list orgs` and `fh list flakes` enumerates orgs and flakes on FlakeHub:
+[`fh list flakes`](#list-flakes), [`fh list orgs`](#list-flakes), and [`fh list versions`](#list-versions) enumerate [flakes], [organizations][orgs], and [flake versions][semver] on FlakeHub, respectively.
 
-```console
-$ fh list orgs
-+-------------------------------------------------------------------------+
-| Organization            FlakeHub URL                                    |
-+-------------------------------------------------------------------------+
-| ajaxbits                https://flakehub.com/org/ajaxbits               |
-| astro                   https://flakehub.com/org/astro                  |
-...
+For each, you can apply the `--json` flag to produce JSON output.
```

Comment:
IMHO this is more appropriate in the automation section below. So, ideally, this would move down there.